include/boards/pico_ice.h

```diff
@@ -129,14 +135,18 @@
 #define PICO_DEFAULT_I2C 0
 #endif
 
+//PICO_PMOD_B1 = 20
 #ifndef PICO_DEFAULT_I2C_SDA_PIN
-#define PICO_DEFAULT_I2C_SDA_PIN 12
```

Comment:
I think there is some plan to reuse the RGB LEDs as I2C pins, might be worth discussing it as it also makes things less convenient by default